ui/lib/home/page/home_page.dart

```diff
@@ -19,13 +25,26 @@ class HomePage extends StatelessWidget {
       ),
       body: Row(
         children: [
+          _buildserverSideBar(context),
           _buildSideBar(context),
           Expanded(child: _buildMain(context)),
         ],
       ),
     );
   }
 
+   Widget _buildserverSideBar(BuildContext context) {
+    return SizedBox.fromSize(
+      size: Size.fromWidth(Sizing.serverSideBarWidth),
+      child: Column(
+        children: [
+          for(final server in serverLabels)
+            Expanded(child: HomeButton(label: server, iconData: Icons.games_outlined,)),
```

Comment:
File an issue to replace HomeButton with a new "ServerButton" widget and add comment.

Reply:
Disregard